CHANGES.rst

```diff
@@ -32,7 +32,8 @@ Internal Changes
 
 - Add ``category-sidebar`` template hook and blocks around category sidebar
   sections (:pr:`5237`, thanks :user:`omegak`)
-
```

Comment:
Please keep the number of empty lines as they were before :)

Reply:
Now there's an empty line between the individual changelog entries - but there should be two blank lines after all of them.